src/reducers/index.js

```diff
@@ -1,21 +1,39 @@
-import { INCREMENT, DECREMENT } from '../actions';
+import { INCREMENT, DECREMENT, RESET, INCREMENTODD, INCREMENTASYNC } from '../actions';
 
 const initialState = {
   count: 0
 }
 
+console.log('This is the initialState.count', initialState.count)
+
 // Our reducer that handles our two action cases:
 // increment and decrement. It receives the state
 // of our redux store, along with an action created
 // by our action creator. What does the reducer
 // need to do with the count in each case?
-export default (state = initialState, action) => {
+export default (state = initialState.count, action) => {
+  console.log('In the reducer yo', action);
   switch (action.type) {
-    case INCREMENT:
-    // Fill in the body of this case
-    case DECREMENT:
-    // Fill in the body of this case
+    case INCREMENT: // Increase by one.
+      return ++state;
```

Comment:
```suggestion
      return {...state, count: state.count +1}
```
Should really try to do this in an immutableway. In the above we're returning a new object, filling it with the old state object's properties and then changing a single property.